### ISSUE TITLE
Druid

### ISSUE THREAD

**Issue by pantsguyapp** [ldtteam/MinecoloniesWiki#735]
No druid worker model image
No druid mention in the tower's pages
Lazy editting and unfinished sentence at the end of the guard page.

**Comment by ravenbuilder934** [comment]
you're perfectly welcome to submit a PR to fix it yourself, you know

**Comment by Talyda** [comment]
Yeah, I don't feel a need to be that polite.
That was damned rude. If you are so much better than the information given, do it yourself.
You constantly open issues here and on the main github, and don't show any basic courtesy - just demands. You are not a special case - you need to show manners. None of us get paid to put up with your crap

**Comment by pantsguyapp** [comment]
> Yeah, I don't feel a need to be that polite. That was damned rude. If you are so much better than the information given, do it yourself. You constantly open issues here and on the main github, and don't show any basic courtesy - just demands. You are not a special case - you need to show manners. None of us get paid to put up with your crap

Yeah, I also don't have time, similar to your situation, but I am just an outsider who isn't even frequent to the mod's discord server, while a wiki developer should be close to the developers themselves and therefore be able to construct a proper wiki page much faster, after seeing something as small as an alternative version of the Guards to not be added even after 2 weeks/months(?), of course I will want t see it immediately, and since I don't want to do bidding with Discord again so I came here instead, and got lashed out by the devs for rudeness?

> "Just make a PR dude"
Fine, I will, if it is what it is.

**Comment by Talyda** [comment]
Firstly, I am not a dev. Never anywhere have I ever implied I am.
Secondly, whatever your expectations are of how the wiki SHOULD be run according to your sense of entitlement, it is run by volunteers. In their spare time. For free.
Thirdly, the druid was added to the guards page ten days ago. The druid was added to the game 17 days ago. I don't know where the hell you think you are finding 2 months.
Yes, damned rude. For the second time.

**Comment by pantsguyapp** [comment]
7 days span, I see, then thats on me for calling you guys out with a fake time span, I have also see now that this mod wiki is written by purely, just volunteers for free, unlike other mainstream wikis, thats all, just going to lower my expectation for the next time.

> that was very rude.

There's no point in bashing the volunteers for something, so you have my apology. Yes, I have also made a bunch of demands on the main page before, to my defense, I tried to limit those issues to only be about actual bugs in the mod, and how it can run better, I can't tell if something is intended, especially when said thing is quite wonky and ineffective (such as citizens AI, but I don't want to go into details), so I just made those issues.
